documentation/smg/10_build_release_process.md

```diff
@@ -55,52 +70,37 @@ Disadvantages:
 
 - slower; have to wait for full build and test pipeline to execute
 
-### Releases
 
-The Jenkins build artifacts are are not accessible outside of STFC. End users will be directed to GitHub to access packaged releases.
 
-A 'deploy' pipeline will be created ([Horace #73](https://github.com/pace-neutrons/Horace/issues/73)). 
+### Hot-fix Releases
 
-This will:
+If a particular fix is required quickly on a target system so that facility users to not lose critical machine time the full build and test process can be bypassed. 
```

Comment:
```suggestion
If a particular fix is required quickly on a target system, the full build and test process can be bypassed so that facility users do not lose critical machine time. 
```